CHANGELOG.md

```diff
@@ -18,7 +18,23 @@ and this project does not adheres to [Semantic Versioning](http://semver.org/).
 
 ## [0.1] - 2014-11-15
 ### Added
-- First version of mex-wholebodymodel, compatible with wholebodyinterface v0.1 . 
+- First version of mex-wholebodymodel, compatible with wholebodyinterface v0.1. 
+
+## [0.2] - 2017-01-25
+### Added
+- New version of mex-wholebodymodel. A complete refactory of the toolbox has been done.
+
+### Renamed
+- WBM-wrappers functions have been renamed, and some of the outputs are listed in a different way.
```

Comment:
Are you going to document which outputs have changed after the merge? Please have mercy of me, I need to migrate another repository (the TU/E) to the new version of mex-wholebodymodel, and checking with trial&error which outputs changed does not seem really enjoyable.